src/frontends/common/src/manager.cpp

```diff
@@ -18,6 +18,15 @@ using namespace ov::frontend;
 class FrontEndManager::Impl {
     std::mutex m_loading_mutex;
     std::vector<PluginInfo> m_plugins;
+    // List of predefined plugins file names (without prefix and suffix)
+    const std::vector<std::string> m_plugins_files_names = {"ir",
```

Comment:
Only the last should plural
```suggestion
    // List of predefined plugin file names (without prefix and suffix)
    const std::vector<std::string> m_plugin_file_names = {"ir",
```

Reply:
Thanks! I decided to rename as `m_plugins_file_names` to make it similar to `m_plugins` and to save meaning "names of plugins" instead of "plugin names"